nautobot/core/views/mixins.py

```diff
@@ -234,6 +235,9 @@ class NautobotViewSetMixin(GenericViewSet, AccessMixin, GetReturnURLMixin, FormV
     serializer_class = None
     table_class = None
     notes_form_class = NoteForm
+    permission_classes = [
+        IsAuthenticated,
+    ]
```

Comment:
Why is this defined here instead of on `SavedViewUIViewSet` specifically? Does it apply to any other UIViewSets?